### PR TITLE
Add ubuntu-23.10 & Arch flists in Micro & Full VM

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Add ubuntu-23.10 & Arch flists in Micro & Full VM

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2504
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2505

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by Mahmoud-Emad — APPROVED** [review]

File: packages/playground/src/weblets/full_vm.vue [hunk 1]
```diff
@@ -176,6 +181,11 @@ const images: VmImage[] = [
     flist: "https://hub.grid.tf/tf-official-vms/ubuntu-18.04-lts.flist",
     entryPoint: "/init.sh",
   },
+  {
+    name: "Arch",
+    flist: "https://hub.grid.tf/petep.3bot/arch_mycelium.flist",
+    entryPoint: "/init.sh",
+  },
```
Comment (AhmedHanafy725) [inline packages/playground/src/weblets/full_vm.vue:188]: it shouldn't be in the fullvm

File: packages/playground/src/weblets/micro_vm.vue [hunk 2]
```diff
@@ -198,11 +198,21 @@ const layout = useLayout();
 const tabs = ref();
 
 const images = [
+  {
+    name: "Ubuntu-23.10",
+    flist: "https://hub.grid.tf/petep.3bot/ubuntu23.10_mycelium.flist",
+    entryPoint: "/init.sh",
```
Comment (AhmedHanafy725) [inline packages/playground/src/weblets/micro_vm.vue:204]: as per the doclerfile, the entrypoint is `zinit`

File: packages/playground/src/weblets/micro_vm.vue [hunk 3]
```diff
@@ -198,11 +198,21 @@ const layout = useLayout();
...
+  },
   {
     name: "Ubuntu-22.04",
     flist: "https://hub.grid.tf/tf-official-apps/threefoldtech-ubuntu-22.04.flist",
     entryPoint: "/sbin/zinit init",
   },
+  {
+    name: "Arch",
+    flist: "https://hub.grid.tf/petep.3bot/arch_mycelium.flist",
+    entryPoint: "/init.sh",
```
Comment (AhmedHanafy725) [inline packages/playground/src/weblets/micro_vm.vue:214]: here too the entrypoint should be `zinit`

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
please try to deploy a vm with each flist and make sure you can access it.

File: packages/playground/src/weblets/full_vm.vue (hunk 1 above)
Comment (zaelgohary) [reply packages/playground/src/weblets/full_vm.vue:188]: Why?

Comment (AhmedHanafy725) [reply packages/playground/src/weblets/full_vm.vue]: cause it doesn't contain a kernel, it just created the file system using dockerfile

Comment (zaelgohary) [reply packages/playground/src/weblets/full_vm.vue]: Removed.

**Review by MohamedElmdary — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**